github/messages_test.go

```diff
@@ -220,3 +220,17 @@ func TestParseWebHook(t *testing.T) {
 		}
 	}
 }
+
+func TestDeliveryID(t *testing.T) {
+	id := "8970a780-244e-11e7-91ca-da3aabcb9793"
+	req, err := http.NewRequest("POST", "http://localhost", nil)
+	if err != nil {
+		t.Fatalf("DeliveryID: %v", err)
+	}
+	req.Header.Set("X-Github-Delivery", id)
+
+	got := DeliveryID(req)
+	if got != id {
+		t.Errorf("DeliveryID(%#v) = %#v, want %#v", req, got, id)
```

Comment:
Drew, the commit message of ffae808934ce7f830c244465e103a28ddda502c1 is "**Fix** formatting of test output". However, both `%#v` and `%q` produce same results for `string` type. It wasn't wrong, this is a mere style change. Just wanted to point that out FYI.